.github/workflows/build-python.yml

```diff
@@ -209,4 +209,38 @@ jobs:
           poetry new ./install-test
           cd ./install-test
           sed -i 's/^\(python *= *\).*$/\1">=3.10,<3.12"/' pyproject.toml
-          poetry add "$(ls ../dist/*.whl)[server]"
\ No newline at end of file
+          poetry add "$(ls ../dist/*.whl)[server]"
+      - name: Create new project, install wheel and import
+        run: |
+          poetry new ./install-run
+          cd ./install-run
+          sed -i 's/^\(python *= *\).*$/\1">=3.10,<3.12"/' pyproject.toml
+          poetry add "$(ls ../dist/*.whl)"
+          poetry run python -c "import giskard"
+  install-pdm:
+    name: "Check if wheel is able to be installed using Poetry"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.1.0
+      - name: Setup PDM
+        uses: pdm-project/setup-pdm@v3
+        with:
+          python-version: '3.10'
+          cache: false
+      - name: Build wheel
+        run: pdm build
+      - name: Create new project, install wheel and import
+        run: |
+          mkdir ./install-run
```

Comment:
I'd still skip the pdm init as we don't need a project to be create just simple venv:

```
python3 -m venv test-install
eval $(pdm venv activate test-install)
```